src/app/core/user/entity-user/entity-user.component.ts

```diff
@@ -0,0 +1,183 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  Input,
+  OnInit,
+  signal,
+} from "@angular/core";
+import { DynamicComponent } from "../../config/dynamic-components/dynamic-component.decorator";
+import { AlertService } from "../../alerts/alert.service";
+import { HttpClient } from "@angular/common/http";
+import { UntilDestroy } from "@ngneat/until-destroy";
+import { SessionSubject } from "../../session/auth/session-info";
+import { Entity } from "../../entity/model/entity";
+import { switchMap } from "rxjs/operators";
+import { environment } from "../../../../environments/environment";
+import { UserAdminService } from "../user-admin-service/user-admin.service";
+import { UserAccount } from "../user-admin-service/user-account";
+import { of } from "rxjs";
+import { MAT_DIALOG_DATA, MatDialogRef } from "@angular/material/dialog";
+import {
+  UserDetailsComponent,
+  UserDetailsAction,
+} from "../user-details/user-details.component";
+
+/**
+ * Display User Account details and configuration related to a given profile Entity.
+ * Can be used as a standalone view or in a dialog.
+ */
+@UntilDestroy()
+@DynamicComponent("UserSecurity")
```

Comment:
```suggestion
@DynamicComponent("EntityUser")
@DynamicComponent("UserSecurity") // for backwards compatibility. Prefer to use "EntityUser"
```

Let's align the component ID with the new name also. To do: add this to core.module registration and update the base-configs